recipes/urlchecker/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "urlchecker" %}
+{% set version = "0.0.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9a887fcc0c999bc1a4157fcb140984f46c3fad597a027aaab607365b4106f84c
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - urlchecker=urlchecker.client:main
+  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - pytest-runner
+  run:
+    - python >=3
+    - requests
+
+test:
+  imports:
```

Comment:
Please add a test for the command line tool:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Added!